tools/configure_test_cluster.py

```diff
@@ -98,23 +98,12 @@ def _initialize_dcos_cli(self):
     def configure_master_settings(self):
         logger.info("Live-customizing mesos master")
         venv_path = venvutil.shared_tools_venv()
-        requirements_file = os.path.join(_tools_dir(), 'requirements.txt')
-        # needs shakedown, so needs python3
-        if sys.version_info < (3,4):
-            venvutil.create_venv(venv_path, py3=True)
-            venvutil.pip_install(venv_path, requirements_file)
-
-            script = os.path.join(_tools_dir(), 'modify_master.py')
-            configure_cmd = ['python', script]
-            venvutil.run_cmd(venv_path, configure_cmd)
-        else:
-            venvutil.create_venv(venv_path)
-            venvutil.pip_install(venv_path, requirements_file)
-            venvutil.activate_venv(venv_path)
+        venvutil.create_dcoscommons_venv(venv_path)
+        venvutil.activate_venv(venv_path)
```

Comment:
Removing python2 path and using new utility function to create our "default" venv, since we actually use this set of packages everywhere currently.